compiler/surface/lexer_pl.cppo.ml

```diff
@@ -136,6 +136,9 @@
 (* Directives *)
 
 #define MR_LAW_INCLUDE "Include"
+#define MR_MODULE_DEF "Module"
```

Comment:
@arcz we're adding modules to Catala, do you want to chime in with correct Polish terms for these?